# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/util-linux-versionsort.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- util-linux-ng-2.14.1/mount/lomount.c.orig	2008-09-10 11:02:43.000000000 +0200
+++ util-linux-ng-2.14.1/mount/lomount.c	2008-11-01 14:39:15.000000000 +0100
@@ -25,9 +25,13 @@
 #include "xmalloc.h"
 #include "realpath.h"
 
+#ifdef __UCLIBC__
+extern int versionsort();
+#else
 #ifndef HAVE_VERSIONSORT
 # include "strverscmp.h"
 #endif
+#endif
 
 #define SIZE(a) (sizeof(a)/sizeof(a[0]))
 
